mifosng-android/src/main/java/com/mifos/mifosxdroid/online/groupdetails/GroupDetailsFragment.kt

```diff
@@ -100,8 +107,29 @@ class GroupDetailsFragment : MifosBaseFragment() {
         binding.btnActivateGroup.setOnClickListener {
             onClickActivateGroup()
         }
+        binding.layoutError.findViewById<Button>(R.id.btnTryAgain).setOnClickListener {
+            reloadOnError()
+        }
+    }
+
+    fun reloadOnError() {
+        sweetUIErrorHandler?.hideSweetErrorLayoutUI(binding.scrollGroupDetails, binding.layoutError)
```

Comment:
Move this line in groupDetailsUiState GroupDetailsUiState.ShowProgressbar state and rename the method loadGroupDetailsAndAccounts()